docs/get-started.asciidoc

```diff
@@ -0,0 +1,275 @@
+////
+Goal of this doc:
+The user is able to successfully see data from their .NET application make it to the Elastic UI via the Elastic Distribution for OpenTelemetry .NET
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+* They have dotnet installed
+////
+
+[[get-started]]
+== Get started
+
+:language: .NET
+:language_lc: dotnet
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+This guide shows you how to use the {distro_name} ("the distro") to instrument your {language} application and send OpenTelemetry data to an Elastic Observability deployment.
+
+[NOTE]
+====
+As an OpenTelemetry SDK, the distro supports sending data to any OpenTelemetry protocol (OTLP) endpoint (https://opentelemetry.io/docs/collector/[OpenTelemetry Collector]), but this guide assumes you are sending data to Elastic.
+====
+
+////
+✅ What the user needs to know and/or do before they install the distro
+////
+[discrete]
+[[get-started-prerequisites]]
+=== Prerequisites
+
+Before getting started:
+
+// Familiarity with OpenTelemetry
+* *Familiarize yourself with the OpenTelemetry SDK.* The distro is a lightweight wrapper around the OpenTelemetry SDK. You should be broadly familiar with the OpenTelemetry SDK concepts and instrumenting applications using the Microsoft diagnostic APIs. If you are not familiar with this, start with the https://opentelemetry.io/docs/languages/net[OpenTelemetry SDK documentation].
```

Comment:
This content was in the getting started guide before I started making edits. I carried over the "Familiarize yourself with the OpenTelemetry SDK" prerequisite to the Java and Node.js getting started guides, too. However, @trentm brought up an interesting point in the Node.js PR:

>I'm concerned that saying a new user needs to familiarize with the OTel SDK before using our distro will somewhat scare them off. The ideal golden path is for users to just install our distro package, set 2 or 3 environment variables to configure it to run with their app, and then see tracing/metrics/logs in their Elastic instance -- no real familiarity with OpenTelemetry required.

For the .NET distro, do users really need to be broadly familiar with the OpenTelemetry SDK, or can they build up that knowledge later if/when they want to customize how they're using the distro?

Reply:
Its a tight rope to walk indeed!

Our distribution are as plug and play as possible yet still exposing the full power of the underlying OpenTelemetry infrastructure. That's part of our value add.

I would refrain from calling us a `wrapper` since it might indicate us masking features. 

Something like:

> The Elastic Distribution is an open extension of the OpenTelemetry SDK. It's goal is to ship an easy to install Observability setup with the right defaults without restricting access to the underlying OpenTelemetry SDK concepts. 
We encourage you to read the https://opentelemetry.io/docs/languages/net[OpenTelemetry SDK documentation] if you need tighter control over your configuration.  

perhaps?